Extension/src/LanguageServer/protocolFilter.ts

```diff
@@ -7,30 +7,60 @@
 import { Middleware } from 'vscode-languageclient';
 import { ClientCollection } from './clientCollection';
 import { Client } from './client';
+import { SourceFileConfiguration, CustomConfigurationProvider } from '../api';
+import { provideCustomConfiguration } from './extension';
+
+function runThenableWithTimeout(promise: Thenable<any>, ms: number): Thenable<any> {
```

Comment:
This should be removed. There is another copy of this in extension.ts which is what is actually being used.